src/app/api/event/route.client.ts

```diff
@@ -0,0 +1,52 @@
+import { Event } from "@prisma/client";
+
+// interface addEventRequest {
+
+// }
+
+// {
+//   EventName: string;
+//   Description: string;
+//   MaxPeople: number;
+//   DateTime: Date;
+// };
+export const addEvent = async (request: {
+  body: { event: Partial<Event> };
```

Comment:
Instead of Partial<Event> I think we should use the Pick keyword in TypeScript so we can make things more specific like this:

type CreateEventInput = Pick<Event, "EventName" | "Description" | "MaxPeople" | "DateTime">